### ISSUE TITLE
Unselectable items for MultiChoice/Checklist

### ISSUE THREAD

**Issue by ahmetb** [abiosoft/ishell#39]
Currently all items in MultiChoice/Checklist are selectable.

If there was a way to specify unselectable items, those could be used as section headers to separate a long list of items. While cycling through options, the cursor wouldn't land on these items, it would just move onto the next item.

**Comment by abiosoft** [comment]
Yeah, I thought about that when I saw it in inquirer.js. On my todo list.
